modules/search/class.jetpack-search.php

```diff
@@ -1762,7 +1762,7 @@ public function track_widget_updates( $option, $old_value, $new_value ) {
 			return;
 		}
 
-		jetpack_tracks_record_event(
+		\Automattic\Jetpack\Tracking::jetpack_tracks_record_event(
```

Comment:
Did you mean to use `tracks_record_event`? Also, why call it statically - it doesn't seem to be a static method?